domains/Settings/SettingsHome/AccountSettings/FindRecords/index.js

```diff
@@ -0,0 +1,210 @@
+import React, { useEffect, useState } from 'react';
+import {
+  ActivityIndicator, Alert, StyleSheet, View
+} from 'react-native';
+import {
+  Button, Headline, IconButton, Text, TextInput
+} from 'react-native-paper';
+
+import { getData, storeData } from '../../../../../modules/async-storage';
+import I18n from '../../../../../modules/i18n';
+import { theme } from '../../../../../modules/theme';
+
+const FindRecords = () => {
+  useEffect(() => {
+    async function setUserInformation() {
```

Comment:
did you check that running an async function in the useEffect doesn't introduce a memory leak?

Reply:
yes, that was the proper way to perform the async within a useEffect from what i found online (here's example): https://javascript.plainenglish.io/how-to-use-async-function-in-react-hook-useeffect-typescript-js-6204a788a435